float32[] angles
bool thresh
bool joint_space_impd
---
bool data
